node-core/node/node.go

```diff
@@ -41,17 +42,18 @@ type node struct {
 	logger log.Logger
 	// registry is the node's service registry.
 	registry *service.Registry
-
-	// TODO: FIX, HACK TO MAKE CLI HAPPY FOR NOW.
-	// THIS SHOULD BE REMOVED EVENTUALLY.
-	types.Node
```

Comment:
rollback command currently fails because this nil interface is called when command try and load CommitMultistore.
Instead we should use the commit multistore provided by consensus service